templates/package/content/generic.tmpl

```diff
@@ -6,7 +6,7 @@
 				<label>{{svg "octicon-terminal"}} {{.locale.Tr "packages.generic.download"}}</label>
 				<div class="markup"><pre class="code-block"><code>
 					{{- range .PackageDescriptor.Files -}}
-curl <gitea-origin-url data-url="{{AppSubUrl}}/api/packages/{{$.PackageDescriptor.Owner.Name}}/generic/{{$.PackageDescriptor.Package.Name}}/{{$.PackageDescriptor.Version.Version}}/{{.File.Name}}"></gitea-origin-url><br>
+curl <gitea-origin-url data-url="{{AppSubUrl}}/api/packages/{{$.PackageDescriptor.Owner.Name}}/generic/{{$.PackageDescriptor.Package.Name}}/{{$.PackageDescriptor.Version.Version}}/{{.File.Name}}"></gitea-origin-url><br class="gt-di">
```

Comment:
It doesn't look good to me ....

I would say that the `<br>` was polluted somewhere else.

Couldn't the pollution be fixed properly?

Reply:
@silverwind 

```
.markup code br,
.markup tt br {
  display: none;
}
```

Why do we need these styles?

Reply:
> Why do we need these styles?

These do come from [original GitHub CSS](https://github.com/sindresorhus/github-markdown-css/blob/5ec832f378e36706bc1af26c65d5f0655ab33ef4/github-markdown.css#L789-L792).